source/entries/tips/how-to-add-mixins.html.erb

```diff
@@ -0,0 +1,18 @@
+---
+
+title: Mixins
+pre_title: How to add
+date: 2017-10-03 20:11 UTC
+layout: recipes
+tags:
+category: tips
+illustration_tag: ../../../images/illustrations/custard.png
+
+---
+
+Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.
```

Comment:
Perhaps set your text editor to wrap lines at 80 characters? It makes multi line paragraphs easy to comment on it Github. I can explain more in person.

Reply:
It doesn't look like this is wrapping?